### PR TITLE
initial docs for docker users

### PR DESCRIPTION
**Pull request by paselem**

### CODE REVIEW

File: docs/30-customize-cluster.md [hunk 1]
```diff
@@ -53,18 +64,11 @@ FROM ubuntu:16.04
 CMD ["R"]
 ```
 
-There is no requirement to be debian based. For consistency with other pacakges it is recommeneded though. Please note though that the container **must be based off a Linux distribution as Windows is not supported**.
-
-### List of tested container images
+For more information and samples on how to build images, deploy them to dockerhub and use them in your cluster please refer to the [Building Containers](./32-building-containers.md) documentation.
 
-The following containers were tested and cover the most common cases for end users.
+There is no requirement to be debian based. For consistency with other pacakges it is recommeneded though. Please note though that the container **must be based off a Linux distribution as Windows is not supported**.
```
Comment (zfengms) [inline docs/30-customize-cluster.md]: typo: pacakges should be packages

File: docs/30-customize-cluster.md [hunk 2]
```diff
@@ -35,6 +35,17 @@ Specifying a docker container is done by updating your cluster.json file. Simply
...
 
+### List of tested container images
+
+The following containers were tested and cover the most common cases for end users.
+
+Container Image | R type | Description
+--- | --- | ---
+[rocker/tidyverse](https://hub.docker.com/r/rocker/r-ver/) | Open source R | Tidyverse is provided by the rocker org and uses a standard version of R developed by the open soruce community. rocker/tidyverse typically keeps up with the latest releases or R quite quickly and has versions back to R 3.1
+[nuest/mro](https://hub.docker.com/r/nuest/mro/) | Microsoft R Open | [Microsoft R Open](https://mran.microsoft.com/open/) is an open source SKU of R that provides out of the box support for math packages, version pacakge support with MRAN and improved performance over standard Open Source R.
+
```
Comment (zfengms) [inline docs/30-customize-cluster.md]: type: pacakge should be package

File: docs/32-building-containers.md [hunk 3]
```diff
@@ -0,0 +1,183 @@
...
+FROM rocker/r-ver
+
+# Install any dependencies required for the R packages
+RUN  apt-get update \
+  && apt-get install -y --no-install-recommends \
+  libxml2-dev
+
+
+# Install the R Packages from CRAN
+RUN Rscript -e 'install.packages(c(jsonlite, httr), dependecies = TRUE)'
```
Comment (zfengms) [inline docs/32-building-containers.md]: typo: dependecies should be dependencies

Comment (paselem) [reply docs/32-building-containers.md]: I have removed that bit of code.

**Pull request merged**